### PR TITLE
RLD-86: Add linter action for projects

### PR DESCRIPTION
**Pull request by scopsy**
## Why?
For better code styling and consistency checks for our codebase.

Fixed current errors with the linter and introduced a GitHub action for checking errors in CI

### CODE REVIEW

File: apps/web/src/App.tsx [hunk 1]
```diff
@@ -39,6 +40,8 @@ const defaultQueryFn = async ({ queryKey }: { queryKey: string }) => {
   return response.data?.data;
 };
 
+console.log('Hi');
```
Comment (djabarovgeorge) [inline apps/web/src/App.tsx:42]: Did the eslint did not worked for you with no-console checks? or did you miss that one? 🙃

**Comment by djabarovgeorge** [comment]
Looks great and great addon I left only a small comment for you 🤟

File: apps/web/src/App.tsx (hunk 1 above)
Comment (scopsy) [reply apps/web/src/App.tsx:42]: 😅  You are very correct hhh, it's a warn level, should be throwing in CI, but it didn't. Good catch!

Comment (scopsy) [reply apps/web/src/App.tsx]: @djabarovgeorge Fixed :)

**Review by djabarovgeorge — APPROVED** [review]

**Pull request merged**